src/test/java/org/apache/commons/imaging/formats/jpeg/xmp/JpegXmpParserTest.java

```diff
@@ -1,3 +1,19 @@
+/*
```

Comment:
Great catch on the missing licence headers! As I'm adding comments to this PR, and I'm just passing by the component to see if it supports something that I need for a project - and am not really familiar with the code base. It may take a bit longer to merge this PR. But if you submit another one just for the headers, we can merge it pretty much immediately ;-)

Reply:
Created #31 which contains only license header corrections.